### PR TITLE
Joint limits in MPC

### PR DESCRIPTION
**Pull request by grizzi**
Add joint limits to MPC.

The limits config are in the task.info file. @jk-ethz, the UR5 task file needs to be updated accordingly.
Waiting for real robots tests for final merge.

### CODE REVIEW

File: moma_controllers/moma_ocs2/src/MobileManipulatorDummyMRT.cpp [hunk 1]
```diff
@@ -97,8 +95,8 @@ int main(int argc, char** argv) {
 
   // initial command
   vector_t initTarget(7);
-  initTarget.head(3) << 0, 1, 1;
-  initTarget.tail(4) << Eigen::Quaternion<scalar_t>(1, 0, 0, 0).coeffs();
+  initTarget.head(3) << 0.5, 0.0, 0.8;
+  initTarget.tail(4) << Eigen::Quaternion<scalar_t>(0.0, 0.7071068, 0, 0.7071068).coeffs();
```
Comment (jk-ethz) [inline moma_controllers/moma_ocs2/src/MobileManipulatorDummyMRT.cpp:99]: Nice one :)

**Review by jk-ethz — APPROVED** [review]
LGTM. I'll change the UR limits, but that can also happen in a separate PR if you prefer to merge now.

**Comment by grizzi** [comment]
Anyway I have to test on the robot and in sim before.

**Comment by grizzi** [comment]
If this is tested now, can we merge?

**Comment by jk-ethz** [comment]
> If this is tested now, can we merge?

Yes

**Pull request merged**